macros/utils/graph/get_compiled_sql_from_node.sql

```diff
@@ -0,0 +1,9 @@
+{% macro get_compiled_sql_from_node(node) %}
```

Comment:
Shouldn't this be generic for any long string value we retrieve? 
Now that the max size is configurable, users can reduce it and values other than 'compiled_query' will fail as well. 
Maybe it's premature and we should do it on another PR, but I think it's easy. No?

Reply:
I agree with you, but I think that maybe it should be another PR 🤔 
Anyhow, as long as we don't document it, it is not really configurable and have less of a chance to happen.

Let me see how much effort it will be to replace every place that uses long string